src/Parser/Core/CombatLogParser.js

```diff
@@ -249,6 +251,29 @@ class CombatLogParser {
         });
       }
     }
+    if (!this.modules.prePotion.usedPrePotion) {
+      results.addIssue({
+        issue: <span>You forgot to use a pre potion before a pull. Using a prepotion can be very effective (even for healers), especially during shorter encounters.</span>,
```

Comment:
More basic info might help some users, e.g.;
`You forgot to use a potion before combat. Using a potion before combat allows you the benefit of two potions in a single fight. A potion such as <ItemLink id={ITEMS.POTION_OF_PROLONGED_POWER.id} /> can be very effective (even for healers), especially during shorter encounters.`